interface/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+#![allow(clippy::arithmetic_side_effects)]
+#![deny(missing_docs)]
+#![cfg_attr(not(test), warn(unsafe_code))]
+
+//! An ERC20-like Token program for the Solana blockchain
+
+use solana_program_error::{ProgramError, ProgramResult};
+use solana_pubkey::Pubkey;
```

Comment:
Typically, with the rustfmt options set at the repo level, uses get condensed into one, ie
```suggestion
use {
    solana_program_error::{ProgramError, ProgramResult},
    solana_pubkey::Pubkey,
};
```

But more importantly, this showed that the new crate isn't being tested or checked -- can you add CI steps to format / lint / test the interface?